src/components/package/show/package-show.js

```diff
@@ -528,6 +527,7 @@ class PackageShow extends Component {
         id="packageShowUsageConsolidation"
         isOpen={sections.packageShowUsageConsolidation}
         onToggle={this.handleSectionToggle}
+        onFilterSubmit={() => {}}
```

Comment:
And why do we need this prop? If it is added for the future implementation, please add the comment with story where it will be implemented. If not this prop could be not required because it looks like `noop` function and you could add it to `defaultProps`